dace/transformation/passes/scalar_to_symbol.py

```diff
@@ -227,7 +227,7 @@ def find_promotable_scalars(sdfg: sd.SDFG, transients_only: bool = True, integer
     interstate_symbols = set()
     for edge in sdfg.edges():
         interstate_symbols |= edge.data.free_symbols
-    for candidate in (candidates - interstate_symbols):
```

Comment:
Without this change the generated cloudsc code fails, I did not investigate it further and according to git blame the change does not stem from me.